fplus-lib/src/core/application/mod.rs

```diff
@@ -132,6 +132,17 @@ impl file::ApplicationFile {
             ..self.clone()
         }
     }
+
+    pub fn kyc_request(&mut self) -> Self {
```

Comment:
```suggestion
    pub fn kyc_request(&self) -> Self {
```

I don't think mut is needed here. We're cloning everything and returning a new Self.